src/component/DoctorList/DocElement.tsx

```diff
@@ -0,0 +1,25 @@
+// map of the list of doctor for index
+import React from "react";
+
+const Doc: React.FC<any> = ({ item }) => {
```

Comment:
We have the type of Doctor, why not use it?
```suggestion
const Doc: React.FC<Doctor> = ({ item }) => {
```